packages/compass-e2e-tests/tests/instance-sidebar.test.ts

```diff
@@ -137,7 +137,7 @@ describe('Instance sidebar', function () {
 
   it('can create a collection and drop it', async function () {
     const dbName = 'test'; // existing db
-    const collectionName = 'my-sidebar-collection';
+    const collectionName = 'a-sidebar-collection';
```

Comment:
why not just search in the sidebar so it is the only one being displayed?

Reply:
ah damn. i didn't know that we can also search by collection name. that sounds much better. I always tried `db.collection` but of course it didn't work.